README.md

```diff
@@ -11,11 +11,17 @@ Codenames is a board/card game where two teams, red and blue, compete against ea
 - Understanding the process of refactoring code to make it more readable and easy to debug
 
 ## Running the Game
-- `python3 main.py`
+ ```sh
+python3 main.py
+ ```
 
 ## Testing the Game
-- `cd tests`
-- `pytest -v`
+```sh
+cd tests
+```
```

Comment:
Lines 21 and 22 are not necessary. The triple backticks are for enclosing multiline code blocks so you don't need to repeat them for each line of code.

Reply:
Sounds good. I did it at first so if you wanted to copy each separately you could but it does look better as one code block.